gatsby-node.js

```diff
@@ -17,7 +17,7 @@
 
   OTHER Instructions
   - For the path alias, you will also need to:
-    - Add "node__contenttype.fields.alias": `PathAlias`, to the mapping in gatsby-config.js
+    - Add "(filter: {relationships: {node__page: {elemMatch: {id: {eq: $id}}}}})_contenttype.fields.alias": `PathAlias`, to the mapping in gatsby-config.js

```

Comment:
It looks like the PathAlias mapping in gatsby-config.js was removed last year, so these instructions no longer appear to be relevant.

Reply:
removed those lines